vs/dosbox.vcxproj

```diff
@@ -276,7 +276,13 @@ IF %ERRORLEVEL% LSS 8 SET ERRORLEVEL = 0</Command>
     </ResourceCompile>
     <PostBuildEvent>
       <Command>%systemroot%\System32\robocopy ..\contrib\resources "$(OutDir)resources" /mir /xf "meson.build" "*.sh"
-IF %ERRORLEVEL% LSS 8 SET ERRORLEVEL = 0</Command>
+IF %ERRORLEVEL% LSS 8 SET ERRORLEVEL = 0
+rd /s /q D:\Emulators\Dosbox\resources
+del D:\Emulators\Dosbox\*.exe
+del D:\Emulators\Dosbox\*.dll
+copy $(TargetDir)$(TargetName)$(TargetExt) D:\Emulators\Dosbox
+copy $(TargetDir)*.dll D:\Emulators\Dosbox
+xcopy $(TargetDir)resources D:\Emulators\Dosbox\resources /E /I</Command>
```

Comment:
What is all this?  I assume you didn't mean to commit this so please remove it.  If it's intentional, put this in a separate commit and explain the purpose.

Reply:
Oh, FFS, not again.